Remove octez node stuff and simplify octez-client stuff for developers (#137)
trilitech/tezos-developer-docs: docs/developing/octez-client/transactions.md

---

> title: Sending transactions

Reviewer: This section is a bit tricky as it's called `Sending transactions`, but we have way more than that here, so maybe `Blockchain interactions` or `Transaction interactions` or just `Transactions with Octez` WDYT @timothymcmackin ?

Author: Hmmm. I like "sending transactions" because it's a task instead of a concept and because it's simple and something that developers need to do to test contracts. Maybe "Interacting with contracts"?

Reviewer: yep, "Interacting with contracts" sounds good

Author: Changed to "Interacting with contracts"

authors: Tim McMackin
lastUpdated: 27 October 2023
---

> Before sending transactions with the Octez client, make sure that you are connected to an RPC node for the network that you want to use.

Reviewer 2: it's not a permanent connection to the node, just a configuration parameter, so maybe better (just a suggestion):
```suggestion
Before sending transactions with the Octez client, make sure that your client is configured to use an RPC node for the network that you want to use.
```

Author: That's good, it makes this more technically accurate -- thanks!
